frontend/src/config/i18n/de/drawings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tools_title": "Form auswählen",
+  "draw_rectangle_hint": "Benutze die Maus, um ein Rechteck aufzuziehen.",
+  "draw_ellipse_hint": "Benutze die Maus, um eine Ellipse aufzuziehen.",
+  "draw_free_line_hint": "Benututze die Maus, um eine freie Linie zu zeichnen.",
+  "draw_bezier_polygon_hint": "Klicken Sie mit der linken Maustaste, um einen Punkt zu setzen. Klicken Sie mit der rechten Maustaste, um die Zeichnung zu beenden.",
```

Comment:
@markus2330 If I remember correctly, we should use informal speech when ("per-du"), correct?

Reply:
Yes, we use "Du", I added it in 0370e63579eb17cbf8cceb1f663a7372fa47fd59